src/Display/Homepage.js

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useRef } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getPosts } from '../redux/slice/postSlice';
+import Movies from '../components/Details';
+import Header from '../components/Header';
+
+import '../index.css';
+
+const Shows = () => {
+  const posts = useSelector((state) => state.post.post);
+  const dispatch = useDispatch();
+
+  const shouldGetPosts = useRef(true);
+  useEffect(() => {
+    if (shouldGetPosts.current) {
+      shouldGetPosts.current = false;
+      dispatch(getPosts());
+    }
```

Comment:
As suggested by the previous reviewer :arrow_down:

# Your Design
![your-design](https://user-images.githubusercontent.com/65233860/222853039-26e57d37-b8fe-4598-a14c-a8e9cbb510ad.png)
# Actual Design
![actual-design](https://user-images.githubusercontent.com/65233860/222853053-87806df6-2061-4704-b621-df0e116d207f.png)

Kindly ensure that your app matches the design in terms of the typography and layout (composition and space between elements). There seem to be some differences currently.

- [ ] For instance, the image is supposed to be at the left with the arrow in the top right corner and the text/title :blush:
- [ ] The shape of the arrow is also different from that of the actual design as shown :+1: